build_tools/cmake/iree_benchmark_suite.cmake

```
@@ -190,11 +190,11 @@ function(iree_benchmark_suite)
   )
 
   # Try to check if the compiler supports the TARGET_BACKEND. If
-  # IREE_HOST_BINARY_ROOT is set, we are using a compiler binary, in which
+  # IREE_HOST_BIN_DIR is set, we are using a compiler binary, in which
   # case we can't check it's supported backends just by looking at this build
```

Nit because you're here

```suggestion
  # case we can't check its supported backends just by looking at this build
```

---

Not worth rerunning CI after fixing this though